compatibility_server/Dockerfile

```diff
@@ -34,5 +34,5 @@ EXPOSE 8888
 # 5 replicas per vm, each replica has 10 workers which allows 50 docker images
 # created at most in the vm.
 CMD cd compatibility_checker && \
-  gunicorn -b 0.0.0.0:8888 -w 2 --worker-class sync --max-requests 20 --max-requests-jitter 10 --timeout 300 \
+  gunicorn -b 0.0.0.0:8888 -w 2 --worker-class sync --max-requests 20 --max-requests-jitter 10 --timeout 400 \
```

Comment:
You could make this change and redeploy to see if it changes anything but it looks like there were ~300 timeouts for >3200 successful checks. It could be that you need more machine resources or something. Did the whitelist change recently?

Reply:
As far as I can tell, the whitelist hasn't been changed. The cluster looks healthy, but that's a bit difficult to determine because of  the DIND setup.

I can scale the cluster a bit higher on machine resources to see, but that'll require updating the deployment specification as well.

Reply:
You can try increasing the timeout and seeing what happens...

Reply:
Follow up: I don't have permissions to update the cluster. Its currently sitting at 200 nodes (matching the deployment spec & ant-affinity policy)

Reply:
I made you an admin of the project. I think that you can make your permissions whatever you want now.